plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/areas/FieldEditorWrapper.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright ${year} Google Inc. All Rights Reserved.
```

Comment:
This file was included by mistake: I initially intended to turn the SDK location into a plain DirectoryFieldEditor, but realized I had some validation that couldn't be incorporated.  So it's not fully functional.  I'll remove it.